### DISCUSSION TITLE
How to set MFXTableView or MFXTableRowCell editable.

### DISCUSSION THREAD

**Discussion by infinite-dev22** [palexdev/MaterialFX#329] [Q&A] [answered]
I want to make a table with editable rows or cells in a specific column. I have tried making a cell editable through creating a ```MFXTableRowCell``` in ```setRowCellFactory()``` and looking for a ```setEditable()``` method on it but it lucks any, ```MFXTableView``` also has no ```setEditable()``` method. Is there a way I can achieve this. Thanks in advance.

**Comment by palexdev** [comment, marked as answer]
Not supported by default, you need a custom cell and implement the feature yourself

**Reply by infinite-dev22** [reply]
Thanks
